pydra/engine/tests/test_shelltask_inputspec.py

```diff
@@ -718,7 +936,22 @@ def test_shell_cmd_inputs_di(tmpdir):
         inputImageFilename=my_input_file,
         input_spec=my_input_spec,
     )
-    assert shelly.cmdline == f"DenoiseImage -i {my_input_file} -s 1 -p 1 -r 2"
+    assert (
+        shelly.cmdline
+        == f"DenoiseImage -i {my_input_file} -s 1 -p 1 -r 2 -o {my_input_file}"
+    )
+
+    # input file name and mask file
+    shelly = ShellCommandTask(
+        executable="DenoiseImage",
+        inputImageFilename=my_input_file,
+        maskImageFilename=my_mask_file,
+        input_spec=my_input_spec,
+    )
+    assert (
+        shelly.cmdline
+        == f"DenoiseImage -i {my_input_file} -x {my_mask_file} -s 1 -p 1 -r 2 -o {my_input_file} {my_mask_file}"
```

Comment:
output should look like (note the brackets):

`f"DenoiseImage -i {my_input_file} -x {my_mask_file} -s 1 -p 1 -r 2 -o [{my_input_file}, {my_mask_file}]"